web/docs/user-guides/run-a-pipeline/README.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_position: 0
+---
+
+# Run a pipeline
+
+:::tip What is a pipeline?
+A [pipeline](../../language/pipelines.md) is a chain of operators that begins
+with a [source](../../operators/sources/README.md), optionally includes
+[transformations](../../operators/sinks/README.md) in the middle, and ends in a
+[sink](../../operators/sinks/README.md).
+:::
+
+## In the app
+
+You run a pipeline in the [app](../../setup-guides/use-the-app/README.md) by
+writing it in the editor and hitting the *Run* button.
+
+The following invariants apply:
+
+1. You must start with a source
+2. The browser is always the sink
+
+The diagram below illustrates these mechanics:
+
+![Pipeline in the Browser](pipeline-browser.excalidraw.svg)
+
+For example, write [`version`](../../operators/sources/version.md) and click
+*Run* to see a single event arrive.
+
+## On the command line
+
+On the [command line](../../command-line.md), run `tenzir <pipeline>` where
+`<pipeline>` is the definition of the pipeline as shown in the examples.
+
+The following invariants apply:
+
+1. If you do not provide source, `read json` will be added
+2. If you do not provide sink, `write json` will be added
```

Comment:
These are not really invariants and this is not complete. For example, we add `save stdout` when the pipeline ends with bytes. But this might be okay, just pointing it out.